openshift/4.0/scripts/oc-sonar-dotnet.sh

```diff
@@ -31,7 +31,6 @@ TARGET=${TARGET:-dev}
 SONAR_PROJECT_KEY=${SONAR_PROJECT_KEY:-pims-api-${TARGET}}
 SONAR_PROJECT_NAME=${SONAR_PROJECT_NAME:-PIMS Backend [${TARGET}]}
 SONAR_URL=${SONAR_URL:-https://sonarqube-3cd915-tools.apps.silver.devops.gov.bc.ca}
-SONAR_TOKEN=${SONAR_TOKEN:-872c1fef72c4d4fff20bb98aacdd0df95e6cc43b}
```

Comment:
The rocketchat and sonar token secrets were not set up correctly in tools which is possibly why this was introduced. I'd rather not have this as I don't like exposing secrets content in github as this repository is not private. Also the build probably SHOULD fail if the secrets aren't set up properly.

Reply:
@devinleighsmith  Yes, I remember making this changes and hardcoding the Sonar key. This however should be store as OC secret. Go ahead and make the changes where needed. Thank you